packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -56,5 +56,6 @@
   "internal_dev_description": "Specifies whether or not to set up internal development mode which provides limited SFDX commands.",
   "refresh_types_text": "Refresh Types",
   "refresh_components_text": "Refresh Components",
-  "force_source_retrieve_display_text": "Retrieve Source from Org"
-}
+  "force_source_retrieve_display_text": "Retrieve Source from Org",
+  "org_browser_enabled_description": "(Beta) Specifies whether to show or hide the Org Browser view when a non-scratch org is set as the default org. See the [documentation](https://forcedotcom.github.io/salesforcedx-vscode/user-guide/org-browser) to learn about the Org Browser."
```

Comment:
@hbc-bindu let me know what you think of this wording. I also may need to update the documentation link depending on feedback from the documentation PR.

Reply:
Maybe we could say 'Specifies whether to display the...'? not sure

Reply:
Hi @brpowell, you could rephrase it as:
Indicates whether the Org Browser view is displayed when the default org is a non-scratch org. See Org Browser [documentation] (https://..........)